README.md

```diff
@@ -93,10 +98,9 @@ public boolean onTouchEvent(MotionEvent event) {
 * channel.toUI() 
```

Comment:
The method mentioned here is incorrect. Please check and correct. Correct in all the places.

Reply:
Done